frontend/packages/kubevirt-plugin/integration-tests/tests/models/virtualMachine.ts

```diff
@@ -39,4 +53,28 @@ export class VirtualMachine extends BaseVirtualMachine {
       timeout,
     );
   }
+
+  async addNodeSelectors(labels: MatchLabels) {
```

Comment:
Is it better to move `addNodeSelectors` to `kubevirtDetailView.ts`?

Reply:
No, scheduling is relevant for Virtual Machine only, it cannot be edited in VMI or Virtual Machine Templates

Reply:
okay.